pkg/fileutil/fileutil.go

```diff
@@ -167,6 +168,11 @@ func CopyTree(src, dest string, uidRange *uid.UidRange) error {
 		return nil
 	}
 
+	fis, err := ioutil.ReadDir(dest)
```

Comment:
That sounds pretty arbitrary. Does `CopyTree` work, when destination exists and is not empty? I don't know what to expect from this function - it is not documented.